test/test-cases/sku-init/sku-init.test.js

```diff
@@ -1,59 +1,45 @@
 const rmfr = require('rmfr');
-const fs = require('fs');
 const path = require('path');
 const spawnSkuScriptInDir = require('../../utils/spawnSkuScriptInDir');
-const { promptForBraidTheme } = require('../../../lib/prompts');
 
-// Skipping test, flaky in CI
 describe.skip('sku init', () => {
```

Comment:
Can we unskip this now?
```suggestion
describe('sku init', () => {
```